### ISSUE TITLE
Version manager

### ISSUE THREAD

**Issue by agarny** [opencor/opencor#37]
As an extension of issue #1, it would be good to be able to track the different versions (nightly builds, beta versions and release versions) of OpenCOR and thus allow the user to go from one version to another.

Indeed, say that the user uses an official version of OpenCOR (version N) and that a nightly build (version N+1) becomes available. The user might then want to test version N+1, but then revert to version N afterwards (because version N+1 is too buggy, he would rather stick to an official version, etc.).

In other words, the user should be able to go forward, as well as backward, when it comes to the different versions of OpenCOR.

**Comment by agarny** [comment]
Although this could be useful, I can't see this ever happening, so closing.

Indeed, this would be too much work for little gain since we would have to support Windows (32/64 bit), Linux (32/64 bit) and OS X (64 bit). At the end of the day, our [download page](http://opencor.ws/downloads/index.html) contains all the official versions of OpenCOR, as well as all the recent snapshots, so someone can always go back to a previous version of OpenCOR using one of those...
